helper/communicator/step_connect.go

```diff
@@ -53,6 +53,7 @@ func (s *StepConnect) Run(state multistep.StateBag) multistep.StepAction {
 			Config:      s.Config,
 			Host:        s.Host,
 			WinRMConfig: s.WinRMConfig,
+			WinRMPort:   s.SSHPort,
```

Comment:
The docs indicate there's a [`winrm_port` option](https://packer.io/docs/templates/communicator.html#winrm_port) which is *probably* what we want here. I'm not sure about the NAT case that @markhuber mentioned.